src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/ChangeOfPartyRequest/CreateCohort/WhenCohortIsCreated.cs

```diff
@@ -137,13 +137,20 @@ public void Then_ChangeOfPartyRequestId_Is_Correct(ChangeOfPartyRequestType requ
             _fixture.VerifyChangeOfPartyRequestId();
         }
 
-        [TestCase(ChangeOfPartyRequestType.ChangeEmployer)]
-        [TestCase(ChangeOfPartyRequestType.ChangeProvider)]
-        public void Then_AssignedToOtherPartyEvent_Is_Emitted(ChangeOfPartyRequestType requestType)
+        [Test]
+        public void And_RequestTypeIsChangeOfEmployer_Then_AssignedToOtherPartyEvent_Is_Emitted()
         {
-            _fixture.WithChangeOfPartyType(requestType);
+            _fixture.WithChangeOfPartyType(ChangeOfPartyRequestType.ChangeEmployer);
             _fixture.CreateCohort();
-            _fixture.VerifyAssignedToOtherPartyEventIsEmitted();
+            _fixture.VerifyAssignedToOtherPartyEventIsEmittedForEmployer();
+        }
+
+        [Test]
+        public void And_RequestTypeIsChangeOfProvider_Then_AssignedToOtherPartyEvent_IsNotEmmitted()
```

Comment:
Not sure - this test gives a good understanding of what we are trying to accomplish. But happy to discuss.